tests/parser/test_parse_inreach.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from ogn.parser.aprs_comment.inreach_parser import InreachParser
+
+
+class TestStringMethods(unittest.TestCase):
+    def test_position_comment(self):
+        message = InreachParser().parse_position("id300434060496190 inReac True")
+        self.assertEqual(message['address'], "300434060496190")
+        self.assertEqual(message['model'], 'inReac')
+        self.assertEqual(message['status'], "True")
```

Comment:
The field 'status' seems to be a boolean. Maybe @acasadoalonso could affirm this or maybe provide more details about the OGINREACH protocol. If it is a boolean then the message should also be a 'true' boolean.

Reply:
Will wait for @acasadoalonso to comment. I would also like to ask him if there are more examples of those beacons.